prow/plugins/dco/dco_test.go

```diff
@@ -225,6 +234,205 @@ Instructions for interacting with me using PR comments are available [here](http
 			removedLabel:   fmt.Sprintf("/#3:%s", dcoNoLabel),
 			expectedStatus: github.StatusSuccess,
 		},
+		{
+			name: "should skip dco check as user is member of the trusted org (all commits signed)",
```

Comment:
should be "all commits non-signed"? or the `commit` needs to be updated otherwise 😄

Reply:
Fixed and updated tests to reflect changes related to labels. PTAL